nf_core/__main__.py

```diff
@@ -380,7 +380,9 @@ def list(ctx, pipeline_dir, json):
 @click.pass_context
 @click.argument("pipeline_dir", type=click.Path(exists=True), required=True, metavar="<pipeline directory>")
 @click.option("-t", "--tool", type=str, metavar="<tool> or <tool/subtool>")
-def install(ctx, pipeline_dir, tool):
+@click.option("-l", "--latest", is_flag=True, default=False, help="Install the latest version of the module")
+@click.option("--force", is_flag=True, default=False, help="Force installation of module if module already exists")
+def install(ctx, pipeline_dir, tool, latest, force):
```

Comment:
We need an option to specify the sha here, e.g. `-s, --sha` as discussed on slack